test/ts/index.tsx

```diff
@@ -4,31 +4,29 @@ interface State extends Hyperapp.State {
   count: number
 }
 
-interface Actions extends Hyperapp.Actions<State> {
-  sub(state: State, actions: ActionCallers, value: number): Partial<State>
-  add(state: State, actions: ActionCallers, value: number): Partial<State>
+interface Actions extends Hyperapp.Actions {
+  sub(value: number): Partial<State>
+  add(value: number): Partial<State>
 }
 
-interface ActionCallers extends Hyperapp.ActionCallers<State, Actions> {}
-
 interface Events extends Hyperapp.Events<State, Actions> {
-  log: (state: State, actions: ActionCallers, value: string) => void
+  log: (state: State, actions: Actions, value: string) => void
```

Comment:
I like how this change provides more types to the `actions` parameter here. Before this change, `actions.sub` wasn't the most helpful:
![image](https://user-images.githubusercontent.com/459878/30845588-e9cc5ef8-a248-11e7-8136-854c06b2b9a5.png)

This change increases the help the intellisense gives:
![image](https://user-images.githubusercontent.com/459878/30845533-b63ee862-a248-11e7-86b3-63586ddabedd.png)

Reply:
Yeah. This is very smart to take the problem from the other side. I am really happy that it solve this issue !